src/refmvs.rs

```diff
@@ -87,13 +87,17 @@ pub struct refmvs_mvpair {
 
 #[derive(Copy, Clone)]
 #[repr(C, packed)]
-pub struct refmvs_block {
+pub struct refmvs_block_unaligned {
```

Comment:
Could we get a comment on here noting why we need this double-struct setup in Rust? Would be good for future reference to note why we're deviating from the C here.

Reply:
Fixed in da6781d0a95e1cc2b35a5c26aea3da80a9566800.